app/jobs/scheduled/problem_checks.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+
```

Comment:
```suggestion
```

Reply:
That is my favourite whitespace character, I'm never letting it go!!!!